src/__fixtures__/data/cmsPortalNewsArticles.ts

```diff
@@ -1,38 +1,42 @@
 export const cmsPortalNewsArticlesMock = [
   {
     __typename: 'Article',
-    id: 'cl3buldda0247riyt6h85cpgc',
+    id: 'id1',
```

Comment:
q: can you remind me why we needed to change these?

Reply:
Oh it was just something I thought would be helpful. I copy/pasted the data from a query I did against my local cms, and I just thought all that long text is a pain to look at. Just thinking ahead to if anyone ever needs to differentiate between items in the array via the id it would just be quicker/easier to look at.

Reply:
gotcha -- i think overall we can definitely improve our mock data and should align on some styles, like what should be as close to the format of prod data (the previous id) and what can be simplified (the current id). for a future eng sync, maybe! cc @gidjin

Reply:
Aligning on some standards of when to pick a production format vs not. I could go either way here so probably worth us talking it out a little bit or just agreeing to one. Would / Could the library you were looking at for seed data help in this situation as well?